exec/cnex/bytecode.c

```diff
@@ -109,7 +109,11 @@ void bytecode_loadBytecode(TBytecode *b, const uint8_t *bytecode, unsigned int l
 
     assert(b != NULL);
 
-    if (memcmp(bytecode, "Ne\0n", 4) != 0) {
+    if (memcmp(bytecode, "#!", 2) == 0) {
+        while (bytecode[i] != '\n' && i < len) i++;
```

Comment:
This needs to compare `i < len` *before* accessing the array. Also, use curly braces even though you've only got an increment statement.

Reply:
Fixed.